demo/src/index.js

```diff
@@ -21,14 +21,14 @@ class Demo extends Component {
                 className="ReactWebChat"
             >
                 <ReactWebChatComponent
-                    url="https://dev.feersum.io/chat_sockjs"
+                    url="https://fe.mtnplay.qa.feersum.io/chat_sockjs"
```

Comment:
Uhm...

Reply:
Please remove. Especially here in an open source repo

Reply:
ah yes thanks

Reply:
i Rewrote the histroy and removed this. just for safety